README.md

```diff
@@ -20,10 +20,18 @@ To build or test the plugin using IntelliJ:
 
 * Get set up with the pre-requisites listed here: http://www.jetbrains.org/intellij/sdk/docs/tutorials/custom_language_support/prerequisites.html
```

Comment:
It looks like the following are steps that users should follow sequentially. If that's true, I recommend making them numbered, rather than bullet point, so that is clearer.